management/prompt.go

```diff
@@ -289,3 +463,18 @@ func guardAgainstPromptTypesWithNoPartials(prompt PromptType) error {
 
 	return fmt.Errorf("cannot customize partials for prompt: %q", prompt)
 }
+
+// ReadRendering retrieves the settings for the ACUL.
+//
+// See: https://auth0.com/docs/api/management/v2#!/Prompts/get_acul
+func (m *PromptManager) ReadRendering(ctx context.Context, prompt PromptType, screen ScreenName, opts ...RequestOption) (c *PromptRendering, err error) {
+	err = m.management.Request(ctx, "GET", m.management.URI("prompts", string(prompt), "screen", string(screen), "rendering"), &c, opts...)
```

Comment:
Do we need to handle err in both ReadRendering and UpdateRendering methods while making Management APIs..?